test/test-server/networking_handlers.go

```diff
@@ -437,3 +458,78 @@ func handleNetworkingPeeringDelete(t *testing.T, id string) http.HandlerFunc {
 		}
 	}
 }
+
+func handleNetworkingTransitGatewayAttachmentGet(t *testing.T, id string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		switch id {
+		case "tgwa-invalid":
+			w.WriteHeader(http.StatusNotFound)
+			return
+		case "tgwa-111111":
+			tgwa := getTransitGatewayAttachment("tgwa-111111", "aws-tgwa1")
+			err := json.NewEncoder(w).Encode(tgwa)
+			require.NoError(t, err)
+		}
+	}
+}
+
+func handleNetworkingTransitGatewayAttachmentsList(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		tgwa1 := getTransitGatewayAttachment("tgwa-111111", "aws-tgwa1")
+		tgwa2 := getTransitGatewayAttachment("tgwa-222222", "aws-tgwa2")
+		tgwa3 := getTransitGatewayAttachment("tgwa-222222", "aws-tgwa3")
```

Comment:
Should this be `tgwa-333333` on the last one to avoid duplicate IDs?

Reply:
Nice catch. Thank you!